renovate.json

```diff
@@ -3,11 +3,11 @@
     "extends": [
       "config:base"
     ],
-    "prHourlyLimit": 1,
     "baseBranches": ["main", "release-2.10", "release-2.9"],
     "postUpdateOptions": [
-      "gomodTidy",
+      "gomodTidy"
     ],
+    "schedule": ["before 9am on Monday"],
```

Comment:
the security update would ignore the schedule, this works only for version update, see https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts